dev-python/configshell-fb/configshell-fb-1.1.29-r1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1
+
+DESCRIPTION="A Python library for building configuration shells"
+HOMEPAGE="https://github.com/open-iscsi/configshell-fb"
+SRC_URI="
+	https://github.com/open-iscsi/configshell-fb/archive/refs/tags/v${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+RDEPEND="
+	dev-python/pyparsing[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	dev-python/urwid[${PYTHON_USEDEP}]
+"
+
+PATCHES=(
+	"${FILESDIR}/${P}-replace-getargspec-1.patch"
+	"${FILESDIR}/${P}-replace-getargspec-2.patch"
```

Comment:
Perhaps concatenate them to save space, or fetch via `SRC_URI`.

Reply:
They are fetched via `SRC_URI` now.